src/geode/geometry/quality.cpp

```diff
@@ -28,8 +28,39 @@
 #include <limits>
 
 #include <geode/geometry/basic_objects/tetrahedron.hpp>
+#include <geode/geometry/mensuration.hpp>
 #include <geode/geometry/vector.hpp>
 
+namespace
+{
+
+    namespace Detail
+    {
+        double constexpr sqrtNewtonRaphson( double x, double curr, double prev )
+        {
+            return curr == prev ? curr
+                                : sqrtNewtonRaphson(
+                                      x, 0.5 * ( curr + x / curr ), curr );
+        }
+    } // namespace Detail
+
+    /*
+     * Constexpr version of the square root
+     * Return value:
+     *	- For a finite and non-negative value of "x", returns an approximation
+     *for the square root of "x"
+     *   - Otherwise, returns NaN
+     */
+    double constexpr sqrt( double x )
+    {
+        return x >= 0 && x < std::numeric_limits< double >::infinity()
+                   ? Detail::sqrtNewtonRaphson( x, x, 0 )
+                   : std::numeric_limits< double >::quiet_NaN();
+    }
+
+    constexpr double SIX_SQRT_2 = 6 * sqrt( 2 );
```

Comment:
```suggestion
    constexpr double SIX_SQRT_2 = 6 * std::sqrt( 2 );
```
you only need that, sqrt is constexpr since C++17